protostar/cairo/short_string.py

```diff
@@ -1,8 +1,24 @@
 CairoShortString = int
 
+LAST_ASCII_CHAR_DEC_VAL = 127
+
 
 def short_string_to_str(value: CairoShortString) -> str:
     """
     Reverse of starkware.cairo.lang.compiler.test_utils.short_string_to_felt
     """
-    return value.to_bytes(length=32, byteorder="big").decode("ascii").lstrip("\x00")
+    return _to_bytes(value).decode("ascii").lstrip("\x00")
+
+
+def is_short_string(value: int) -> bool:
+    if value > int.from_bytes(b"\xff" * 31, "big"):
+        return False
+
+    for byte in _to_bytes(value):
+        if byte > LAST_ASCII_CHAR_DEC_VAL:
+            return False
```

Comment:
Should we be also checking that it's in printable character range? That is >= 32 I think?

Reply:
I think we should be consistent with cairo, which states that the short string consists of ascii characters. I don't think they limit ascii characters somehow

Reply:
Right, but how would decoding work if someone passed `0` for example? This would get decoded to `null`. Or maybe it's impossible for values like this to show up?

Reply:
It's possible to panic with such values
In case of something like `assert(1 == 2, 20)` protostar will print
```
Test failed with data: 
[20] (integer representation)
['\x14'] (short-string representation)
```

Reply:
so we could limit it to a printable character range, but at the same time some numbers can represent printable ascii chars, not carrying any meaningful information at the same time
like `16705` will be decoded to `'AA'`

Reply:
I guess we can leave it as is, but it's not a strong opinion